frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx

```diff
@@ -45,20 +47,20 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
     'cluster',
   );
   React.useEffect(() => {
-    watchK8sResource(cephClusterResource);
     watchK8sResource(SubscriptionResource);
+    watchK8sResource(ocsResource);
     return () => {
-      stopWatchK8sResource(cephClusterResource);
       stopWatchK8sResource(SubscriptionResource);
+      stopWatchK8sResource(ocsResource);
     };
   }, [watchK8sResource, stopWatchK8sResource]);
 
   const infrastructurePlatform = getInfrastructurePlatform(infrastructure);
-
-  const cephCluster = _.get(resources, 'ceph');
-  const cephClusterLoaded = _.get(cephCluster, 'loaded', false);
-  const cephClusterData = _.get(cephCluster, 'data') as K8sResourceKind[];
-  const cephClusterName = getName(_.get(cephClusterData, 0));
+  const ocs = resources?.ocs;
+  const ocsLoaded = ocs?.loaded || false;
+  const ocsError = ocs?.loadError;
+  const ocsData = ocs?.data;
+  const ocsName = getName(ocsData?.[0]);
```

Comment:
We need to select the one that isn't in Ignored Phase and use that item's name. 
```
    const cluster =  ocs.find(
      (item: OCSStorageClusterKind) => item.status.phase !== 'Ignored',
    );
   const ocsName = getName(ocs);
```
https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/features.ts#L73-L75

Reply:
addressed